src/logic/treeherder.ts

```diff
@@ -3,7 +3,7 @@ import { Framework, TimeRange } from '../types/types';
 
 // This file contains functions to request the Treeherder API
 
-export const treeherderBaseURL = 'https://treeherder.mozilla.org';
+export const treeherderBaseURL = 'http://localhost:8000';
```

Comment:
For testing purposes, will be removed later.

Reply:
Don't forget to change it back :-) unfortunately this prevents me from testing with the deploy preview :/

Reply:
I haven't merged the backend changes yet, since I want to do some more local tests.